rules/aws/persistence_route_53_domain_transfer_lock_disabled.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/05/10"
+maturity = "development"
+updated_date = "2021/05/10"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = "Identifies when a transfer lock was removed from a Route 53 domain."
+false_positives = ["Unknown"]
```

Comment:
```suggestion
false_positives = [
    """
    A domain transfer lock may be disabled by a system or network administrator. Verify whether the user
    identity, user agent, and/or hostname should be making changes in your environment. Activity from unfamiliar
    users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the
    rule.
    """,
]
```

Reply:
may need to re-lint this